app/src/test/java/com/kickstarter/viewmodels/PledgeFragmentViewModelTest.kt

```diff
@@ -374,18 +463,116 @@ class PledgeFragmentViewModelTest : KSRobolectricTestCase() {
     fun testTotalAmountWithoutShippingRules() {
         val environment = environmentForShippingRules(ShippingRulesEnvelopeFactory.emptyShippingRules())
         setUpEnvironment(environment)
-        this.totalAmount.assertValue("$20")
+        this.totalAmount.assertNoValues()
     }
 
     @Test
     fun testTotalAmountWithoutShippingRules_WhenStepperChangesValue() {
         val environment = environmentForShippingRules(ShippingRulesEnvelopeFactory.emptyShippingRules())
         setUpEnvironment(environment)
-        this.totalAmount.assertValue("$20")
+        this.totalAmount.assertNoValues()
```

Comment:
Isn't the `totalAmount` supposed to emit even when we don't have shipping rules?

Reply:
Not if the shipping rules are empty on a project that is shippable.